### ISSUE TITLE
Key facts bullet pattern

### ISSUE THREAD

**Issue by ericlamarca** [nhsuk/nhsuk-service-manual-backlog#61]
We considered the medicines Key facts pattern as an alternative to our standard bullet list at the Style Council meeting on 3 October. 

Reshma is checking the research around this pattern.

Potentially to come back to Style Council of 30 October.

Mark Robinson is looking at testing this in health literacy.

**Comment by sarawilcox** [comment]
We have quite a lot of content in 'Social care and support guide' where bullets don't have a lead-in line. Are we aligning with GDS, or leaving decisions open on this? The bullets start with a lower-case letter, so it almost looks like the lead-in line just isn't showing.
Example: https://www.nhs.uk/conditions/social-care-and-support-guide/practical-tips-if-you-care-for-someone/how-to-move-lift-and-handle-someone-else/
Are we happy with this?

**Comment by sineadgalvin** [comment]
Not sure about the other pages, Sara, but I feel like those H3s (e.g. In the bedroom) shouldn't be H3s at all and should instead act as the lead-in line. It doesn't look like that page had been subbed, either, so that might go some way towards explaining the problem?

**Comment by sarawilcox** [comment]
Thanks @sineadgalvin. I'm following this up with the medicines team and Kathryn (who worked on the social care pages) to see if we have any evidence for an alternative bullet point format.

**Comment by sineadgalvin** [comment]
OK, thanks @sarawilcox. Another thing that occurred to me is that it's possibly not particularly good for accessibility in terms of headings hierarchy, from what I remember?

**Comment by JackMatthams** [comment]
We agreed to include guidance on an alternative, full sentence bullet list format on the formatting and punctuation page in response to the medicines team posting evidence of their "Key facts" helping users. 

We agreed to the following additions:

"If your user research shows that it helps your users, you can use full sentence bullet points. For example, our medicines information uses full sentence bullet points for key facts.

Example:

Key facts
- Paracetamol takes up to an hour to work.
- The usual dose of paracetamol is one or two 500mg tablets at a time.
- Do not take paracetamol with other medicines containing paracetamol.
- Paracetamol is safe to take in pregnancy and while breastfeeding, at recommended doses.
- Brand names include Disprol, Hedex, Medinol and Panadol. 

Each bullet point should be distinct information. It should be a full sentence, starting with a capital letter and ending with a full stop. There should be no lead in line."

**Comment by bencullimore** [comment]
### Nested bullet points 

On the NHS website, I've noticed a few instances of nested bullet points that aren't clearly defined - we don't reference them in the service manual and I can't find any information on them in the GDS design system. 

For example
![Screenshot 2020-11-25 at 09 15 36](https://user-images.githubusercontent.com/16209425/100330273-ab5d9880-2fc6-11eb-8a83-eabca4f331bc.png)


Not having clearly defined nested bullets could effect the way a user interprets the information - assumption being that they might not realise that the nested points are a subset of the point above. 

Standard web practice is to clearly indent the nested bullets by around 4 characters of the parent along with restyling the 'dot' of the bullet point. 

For example   
![](https://s3.amazonaws.com/webucator-how-tos/419.png)

It would be good for any content designers who are working on content that requires nested bullet points to test assumptions around this.

**Comment by sarawilcox** [comment]
I think we'd steer clear of 2 levels of bullet points. They tend to be used in content that is over-long and complex.

**Comment by bencullimore** [comment]
@sarawilcox do we have guidance suggesting we stay clear in the manual?

**Comment by sarawilcox** [comment]
No, I don't think so. Shall I share in the NHS.UK content channel and see if we can get more insight into this one?

**Comment by mattbancroft23** [comment]
If all the main bullets were got rid of then the "sub-bullets" would be OK I think.

Unfortunately I subbed this at the time but was asked to only read it for typos so did not consider the bullet styling.

**Comment by marklegassick** [comment]
I'm currently working on some baby and child health pages which are very long and, as yet, have not been transformed. But one thing I think that's worth raising is the use of bullet points on these pages. In keeping with the rest of the page, they are also overly long to the point where they should probably be paragraphs rather than bullet points. We need to give some thought to how long bullet points should be - probably as short as possible but it might be worth agreeing limiting them to a set number of words or lines. Anything over this should be in normal paragraph format otherwise the benefits of using bullet points, such as clarity and style, are lost. See this page as an example: https://www.nhs.uk/conditions/pregnancy-and-baby/solid-foods-weaning/

**Comment by sarawilcox** [comment]
The medicines team is A/B testing their information on how to cope with side effects. The current format sometimes involves long bullet points. We'll review bullet points again when they've had a chance to review their findings and decide on the way forward.

**Comment by Tosin-Balogun** [comment]
# What
App icon list is used on a series of better health products which is co-owned by the NHS and Office of health improvement and disparities (OHID).

These products are:
- [Better health](https://www.nhs.uk/better-health/quit-smoking/)
- [Every mind matters](https://www.nhs.uk/every-mind-matters/mental-health-issues/anxiety/)
- [Healthier families](https://www.nhs.uk/healthier-families/food-facts/healthier-food-swaps/)
- [Start for life](https://www.nhs.uk/start-for-life/baby/weaning/how-to-start-weaning-your-baby/)

Screenshot example
<img width="1278" alt="Screenshot 2024-02-12 at 17 27 02" src="https://github.com/nhsuk/nhsuk-service-manual-community-backlog/assets/71393402/759703b9-e31d-4682-ab08-40c2fa072ad2">

## Why
These app icon list are used in multitude of ways to help orient users around dense content. These are content blocks styled to resemble bullet points with an icon replacing the dot.

## Additional details
In the usability testing session I observed users gravitated towards these icon list, often slowing down to read the content details within them compared to other content blocks on the page. 

My observation is that users are able to more easily digest the content because they are broken down into smaller chunks and are separated into bullet points using the icon as a decoy to create some separation of what would have otherwise been a large content block.

I have added a screenshot of the research summary slide deck.

<img width="1166" alt="Screenshot 2023-10-31 at 18 25 46" src="https://github.com/nhsuk/nhsuk-service-manual-community-backlog/assets/71393402/517e9bda-6926-4ec8-9bad-3dda95edc3a6">

These have also tested well when used on Every mind matters wellbeing tips section, having similar effects where the user slows down to digest the tips being offered. These are collection from user feedback taken from a series of usability testing sessions conducted with 20 users in March 2021 by @misakihata and Zoonou (agency)

#### Screen reader behaviour
From what I can gather, the image is entirely ignored and skips to the sub-heading for each content block. This might be because of the presence of a sub-header, it might be repeated content if the icon is described.
<img width="1449" alt="Screenshot 2024-02-12 at 17 21 09" src="https://github.com/nhsuk/nhsuk-service-manual-community-backlog/assets/71393402/4522c245-f0e2-4d29-9dd7-18d248f198c1">

### Related issues
#61 #4

**Comment by sarawilcox** [comment]
We've had a query about bulleted lists. Is our position that a list should have at least 2 bullet points and to rewrite your content as a sentence if it's only 1? 

We have some care cards with a single bullet. (The format forces use of a bullet due to having a lead-in as heading.) And sometimes external links to more info are a single bullet. Also possibly some Do and Don't lists.

But it may be worth specifying that in standard body text, we should write content as a sentence rather than have 1 bullet.
